trace-dispatcher/src/Cardano/Logging/Tracer/Forward.hs

```diff
@@ -12,20 +15,20 @@ module Cardano.Logging.Tracer.Forward
   ) where
 
 import           Codec.CBOR.Term (Term)
-import           Control.Concurrent (threadDelay)
+import           Codec.Serialise (Serialise (..))
 import           Control.Concurrent.STM.TBQueue (TBQueue, newTBQueueIO,
                      writeTBQueue)
 import           Control.Exception (SomeException, try)
-import           Control.Monad (forever)
 import           Control.Monad.IO.Class
 import           Control.Monad.STM (atomically)
-import           Control.Tracer (contramap, stdoutTracer)
+import           GHC.Generics (Generic)
+
+-- Temporary solution, to avoid conflicts with trace-dispatcher.
+import "contra-tracer" Control.Tracer (contramap, stdoutTracer)
```

Comment:
Why is this necessary? Can't trace dispatcher use the arrow based lib? Do we need (yet another) tracer here anyway?